pep-0463.txt

```diff
@@ -3,12 +3,41 @@ Title: Exception-catching expressions
 Version: $Revision$
 Last-Modified: $Date$
 Author: Chris Angelico <rosuav@gmail.com>
-Status: Draft
+Status: Rejected
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 15-Feb-2014
 Python-Version: 3.5
 Post-History: 20-Feb-2014, 16-Feb-2014
+Resolution: https://mail.python.org/pipermail/python-dev/2014-March/133118.html
+
+
+Rejection Notice
+================
+
+I want to reject this PEP. I think the proposed syntax is acceptable given
```

Comment:
If you could say "From https://mail.python.org/pipermail/python-dev/2014-March/133118.html:" and then put the entire blockquote in `"""` then I think this PR will be ready to go.